lib/apia/open_api/objects/path.rb

```diff
@@ -109,7 +109,10 @@ def add_scopes_description
 
           @spec[:security].each do |auth|
             auth.each_key do |key|
-              scope_prefix = @spec[:components][:securitySchemes][key][:"x-scope-prefix"]
+              auth_scheme = @spec[:components][:securitySchemes][key]
```

Comment:
you could call `symbolize_keys!` or probably `deep_symbolize_keys` to forcibly convert they keys to symbols. In this loop is likely not the best place to do that. Perhaps when additions constructed in the Specification `initialize` method. That way it would be standard for any code that accesses it.

Reply:
Thank you! on lookup it seems like that's a rails specific function so I've settled on using `transform_keys` :bow: